.dc-dev/.django.env

```diff
@@ -2,3 +2,4 @@ ASKUPENV=dev
 DJANGO_SETTINGS_MODULE=config.settings.dev
 PGPASSWORD=pass
 LOADMOCKUPDATA=1
+CELERYBEAT_PID_FILE=/var/run/celerybeat.pid
```

Comment:
Why do we care about celerybeat_pid_file?

Reply:
Otherwise it creates the pid file in the repository root as it's mounted into the celery container and it leaves this file in the repository root in between the container runs - what causes container run fail because of existent pid file.
If pid file is stored inside the container fs - it's going to be automatically deleted on the container stop and created on it's start.